### PR TITLE
Add possibility to pass a parameter to the confirmAction

### PR DESCRIPTION
**Pull request by octronic**
Usage: 

``` javascript
{{#confirm-extension 
  confirmAction="confirm" 
  questionText='Do you really want to destroy the planet?'
  confirmText='Yes' 
  declineText='No' 
  actionParam="I need this param"}}
  click me!
{{/confirm-extension}}
```

### CODE REVIEW

**Comment by thriqon** [comment]
There should be documentation on this feature in the Readme... Probably it's enough to just extend the basic example showcasing the `actionParam`:

``` handlebars
{{#confirm-extension
  questionText='Do you really want to destroy the planet?'
  confirmText='Yes' declineText='No'
  confirmAction='destroyPlanet' actionParam='earth'}}
```

File: addon/components/confirm-extension.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default Ember.Component.extend({
     },
     confirm: function() {
       this.set('confirmMode', false);
-      this.sendAction('confirmAction');
+      this.sendAction('confirmAction', this.get('actionParam'));
```
Comment (thriqon) [inline addon/components/confirm-extension.js]: I think `actionParam` is misunderstandable, as the direct connection between `confirmAction` and `actionParam` is missing. Maybe `confirmActionParam` would be a better choice?

**Comment by octronic** [comment]
Renamed from `actionParam` to `confirmActionParam` and added info about its usage in the Readme.

**Comment by thriqon** [comment]
Looks good to me :+1:

**Comment by octronic** [comment]
PR gets closed for now since another solution which better fits the "data down, actions up" paridigm of ember could be found.

If someone else would like to have the `confirmActionParam` this PR may still be merged.

**Pull request closed without merging**